packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -1,23 +1,45 @@
 const path = require('path');
 const { getCwd } = require('@hubspot/cli-lib/path');
-const { PROJECT_COMPONENT_TYPES } = require('@hubspot/cli-lib/lib/constants');
+const {
+  PROJECT_COMPONENT_TYPES,
+  PROJECT_PROPERTIES,
+} = require('@hubspot/cli-lib/lib/constants');
 const { promptUser } = require('./promptUtils');
 const { fetchJsonFromRepository } = require('@hubspot/cli-lib/github');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
 
 const i18nKey = 'cli.lib.prompts.createProjectPrompt';
 
-const createTemplateOptions = async () => {
+const hasAllProperties = projectList => {
+  return projectList.every(config =>
+    PROJECT_PROPERTIES.every(p =>
+      Object.prototype.hasOwnProperty.call(config, p)
```

Comment:
I received an `eslint no-prototype-builtins` error when attempting to use `config.hasOwnProperty`. This is the generally accepted workaround.